src/CMakeLists.txt

```diff
@@ -221,6 +221,7 @@ ncnn_add_layer(Power)
 ncnn_add_layer(PReLU)
 ncnn_add_layer(Proposal)
 ncnn_add_layer(Reduction)
+ncnn_add_layer(StatisticsPooling)
```

Comment:
new layer should always be appended at the end of this list
Because the layer enum index is generated from this order. Inserting would break binary compatibility.

Reply:
OK, I will modify that.

Reply:
Do I need to recommit a version.